tasks/add-unique-granuleID/src/index.ts

```diff
@@ -0,0 +1,57 @@
+import Logger from '@cumulus/logger';
+import { Context } from 'aws-lambda';
+
+import { constructCollectionId } from '@cumulus/message/Collections';
+import { generateUniqueGranuleId } from '@cumulus/ingest/granule';
+import { CumulusMessage, CumulusRemoteMessage } from '@cumulus/types/message';
+import { CumulusMessageWithAssignedPayload, runCumulusTask } from '@cumulus/cumulus-message-adapter-js';
+
+import { HandlerEvent, HandlerOutput } from './types';
+
+const log = new Logger({ sender: '@cumulus/add-unique-granule-id' });
+
+/**
+ * If no producerId is assigned,
+ * assign uniqueIds to the granules in the payload,
+ * preserving the original ID in 'producerGranuleId'.
+ *
+ * @param event - input from the message adapter
+ * @returns the uri of the pan
```

Comment:
More lies?

Reply:
🤦🏻 A bit of unfortunate templating cruft.   e3765714ff1.  Good catch.